src/e2e/resources/data/InstructorSessionIndividualExtensionPageE2ETest.json

```diff
@@ -21,6 +21,12 @@
       "name": "Programming Language Concepts",
       "institute": "TEAMMATES Test Institute 1",
       "timeZone": "Asia/Singapore"
+    },
+    "course2": {
+      "id": "tm.e2e.ISesIe.CS2105",
+      "name": "Programming Language Concepts",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "Asia/Singapore"
```

Comment:
could I check why a new course was added? looking at the E2E test, this new course2 does not seem to be used

Reply:
@cedricongjh Thanks for providing your comments, let me check and get back on this. ty.,

Reply:
hey @suneelval, any updates on this?

Reply:
@cedricongjh Thanks for follow up on this issue, removed mentioned unnecessary mock data.